package.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+project=$(cat package.json | jq -r .name)
+version=$(cat package.json | jq -r .version)
+iteration="$TRAVIS_BUILD_NUMBER.git$(git rev-parse --short HEAD)"
```

Comment:
Yeah, but, I don't really want someone to push a bunch of packages from local. I want that controlled by CI.

I'll add a guard statement at the top that exits the script if travis_build_number is not defined.